classes/settings_provider.php

```diff
@@ -58,6 +58,11 @@ class settings_provider {
      */
     const USE_SEB_CLIENT_CONFIG = 4;
 
+     /**
+      * The filearea itemid which is used for saving the module template.
+     */
+    const SEB_CONFIG_FILE_ITEMID = 111;
```

Comment:
Why is it 111?

Reply:
To remove the magic numbers that were littered throughout the code base. 111 could just be 1.
Specifically not 0 as I didn't want the file itemid to be equated to 0/false anywhere.

Reply:
Maybe pick generic name for that not related to seb config file as it could change in the future. Why not to use $cm->id? Not saying that it should be, just asking...